### ISSUE TITLE
Improve default configuration accessibility

### ISSUE THREAD

**Issue by nikp123** [matrix-org/matrix-bifrost#275]
**Describe the bug**
When using the NEDB as the database, the default config is quite vague about the way that the ``connecitonString`` should be written.

**To Reproduce**
1. Open ``config.yaml``
2. Change database type to ``nedb``
3. Change connection string to ``/data/``
4. Launch bifrost
5. Bifrost fails to launch due to it expecting a ``nedb:///data`` instead

**Expected behavior**
Being able to figure this out without diving into the Program.js file itself

**Desktop (please complete the following information):**
 - Irrelevant 

**Smartphone (please complete the following information):**
 - Irrelevant

**Additional context**
Add a simple comment like: 
 ```
 When using NEDB, in "connectionString" you will need to specify a **directory** that stores the database files in the following format:
 nedb:///pathfromroot
```

**Comment by nikp123** [comment]
Apologies if doesn't fit the "bug" criteria, but is definitely a usability issue.

**Comment by Half-Shot** [comment]
Agreed. The current config leaves much to be desired. I suspect we will want to:

- See if we can trim down some of the config options.
- Specify seperate sample configs for XMPP.js and libpurple.
